platform/qt/resources/common.qrc

```diff
@@ -1,6 +1,6 @@
 <RCC>
     <qresource prefix="/">
-        <file alias="icon.png">../../../common/icon.png</file>
+        <file alias="icon.png">../../../common/mb-icon-blue-square.png</file>
```

Comment:
@brunoabinader @tmpsantos, is this resource file needed anywhere? Would it be possible to use an SVG like the other resources in this file instead of hard-coding a specific resolution?